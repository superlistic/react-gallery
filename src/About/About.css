.about {
  display: flex;
  flex-direction: column;
  /* justify-content: center;    */
  align-items: center;
  margin: 1rem 0;
}

.about__title {
  font-size: 2rem;
  /* margin: 1rem 0; */
  margin-bottom: 2rem;
}

.about__image {
  max-height: 60vh;
}
.about__description {
  margin: 2rem 0;
}
