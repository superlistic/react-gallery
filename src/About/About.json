{
    "title":"About us",
    "description": "blalblabla",
    "picture": "https://24slides.com/presentbetter/wp-content/uploads/2018/12/company-profile-presentation-tips.jpg"
}
