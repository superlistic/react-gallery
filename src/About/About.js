import React from 'react';
import json from './About.json';
import '../main.css';
import './About.css';

const About = () => {
  return (
    <div className="about">
      <h1 className="about__title">{json.title}</h1>
      <img className="about__image" src={json.picture} alt="boardmeeting" />
      <p className="about__description">{json.description}</p>
    </div>
  );
};

export default About;
