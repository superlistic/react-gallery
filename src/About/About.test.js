import React from 'react';
import renderer from 'react-test-renderer';

import About from './About';

test('renders elements', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
