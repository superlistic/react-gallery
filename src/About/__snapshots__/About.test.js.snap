// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders elements 1`] = `
<div>
  <h1>
    About us
  </h1>
  <p>
    blalblabla
  </p>
  <img
    src="https://24slides.com/presentbetter/wp-content/uploads/2018/12/company-profile-presentation-tips.jpg"
  />
</div>
`;
